openfifth/aspen-discovery
# Social media links granularity

This adds the option to choose which of the social media link will appear for your library system, rather than having all of them appear or not.

Test plan: 
1. Navigate to "aspen-administration -> library systems -> {edit default library}" 
2. search for "shar" and enable "Show Sharing To External Sites" -> SAVE
3. Do a search and notice that share links appear under each result
4. Navigate to the above and search for "shar" again.
5. Test the check-boxes for "Show Sharing Link To {x}"
6. After modifying each, check that they appear/don't appear correctly upon a catalogue search.
